src/Ocelot/Configuration/SecurityOptions.cs

```diff
@@ -1,43 +1,17 @@
 ﻿using System.Collections.Generic;
-using System.Linq;
-using NetTools;
 
 namespace Ocelot.Configuration
 {
     public class SecurityOptions
     {
-        public SecurityOptions(List<string> allowedList, List<string> blockedList, bool excludeAllowedFromBlocked = false)
+        public SecurityOptions(List<string> allowedList, List<string> blockedList)
         {
-            IPAllowedList = new List<string>();
-            IPBlockedList = new List<string>();
-            ExcludeAllowedFromBlocked = excludeAllowedFromBlocked;
-
-            foreach (var allowed in allowedList)
-            {
-                if (IPAddressRange.TryParse(allowed, out var allowedIpAddressRange))
-                {
-                    var allowedIps = allowedIpAddressRange.AsEnumerable().Select(x => x.ToString());
-                    IPAllowedList.AddRange(allowedIps);
-                }
-            }
-
-            foreach (var blocked in blockedList)
-            {
-                if (IPAddressRange.TryParse(blocked, out var blockedIpAddressRange))
-                {
-                    var blockedIps = blockedIpAddressRange.AsEnumerable().Select(x => x.ToString());
-                    IPBlockedList.AddRange(blockedIps);
-                }
-            }
-
-            if (ExcludeAllowedFromBlocked)
-            {
-                IPBlockedList = IPBlockedList.Except(IPAllowedList).ToList();
-            }
+            IPAllowedList = allowedList;
+            IPBlockedList = blockedList;
```

Comment:
There are no `null`-checks!

Reply:
This is the original version of the file, without any checks.
I can modify it with this code if needed
```
IPAllowedList = allowedList ?? new List<string>();
IPBlockedList = blockedList ?? new List<string>();
```

Reply:
We are able to improve the code quality constantly.
Yes, please, add null-checks!